docs/data-sources/service.md

```diff
@@ -0,0 +1,204 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "encore_service Data Source - terraform-provider-encore"
+subcategory: ""
+description: |-
+  Encore provisioned service information
+---
+
+# encore_service (Data Source)
+
+Encore provisioned service information
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The name of the Encore resource
+
+### Optional
+
+- `env` (String) The environment of the Encore resource. Defaults to the provider environment
+
+### Read-Only
+
+- `aws_fargate_task_definition` (Attributes) (see [below for nested schema](#nestedatt--aws_fargate_task_definition))
+- `cluster_ip` (Attributes) (see [below for nested schema](#nestedatt--cluster_ip))
+- `gcp_cloud_run` (Attributes) (see [below for nested schema](#nestedatt--gcp_cloud_run))
+- `k8s_container` (Attributes) (see [below for nested schema](#nestedatt--k8s_container))
```

Comment:
Should these be in a sub-object that's a list? So we could represent multiple deployments?

Reply:
multiple deployments of the same service?

Reply:
Yeah, for multi-region use cases etc